napari/utils/events/evented_model.py

```diff
@@ -5,11 +5,13 @@
 from typing import Any, Callable, ClassVar, Dict, Set, Union
 
 import numpy as np
-from pydantic import BaseModel, PrivateAttr, main, utils
+from extra_pydantic import BaseModel, ModelMetaclass
```

Comment:
👀
just want to flag that this is change that effectively influences a huge surface area of napari.  (I know that this is, of course, the whole point of the `extra_pydantic` ...)  but inasmuch as it starts showing up in stack traces, it would be good for other core devs to have a sense for what it does, and what it could possibly affect.